### ISSUE TITLE
Should X-UA-Compatible still be used in default ! template?

### ISSUE THREAD

**Issue by Maxxiii** [emmetio/expand-abbreviation#28]
When using "!" we got template with this tag:
`<meta http-equiv="X-UA-Compatible" content="ie=edge">`
For what I heard, this tag is now optional. For example [this page](https://stackoverflow.com/questions/6771258/what-does-meta-http-equiv-x-ua-compatible-content-ie-edge-do) linking to Microsoft page for IE11, that says:
"Use the HTML5 document type declaration to enable edge mode: <!doctype html>"
In other words, by using <!doctype html> tag we already enabling edge mode, and all previous IE versions are no longer supported by Microsoft.
And in Edge this tag is deprecated (quote from same page):
"As we announced in August 2013, we are deprecating document modes as of IE11"
So, is there some reason to still use this tag in default template?

**Comment by Maxxiii** [comment]
Turned out it's already disabled in recent commit. Thank you and sorry for disturbance.
